networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc

```diff
@@ -16,6 +16,25 @@ Red Hat supports the Kubernetes NMState Operator in production environments on b
 
 Before you can use NMState with {product-title}, you must install the Kubernetes NMState Operator.
 
+[NOTE]
```

Comment:
The two notes in this section still feel redundant. They're both communicating the same limitation in slightly different ways. If one is unclear on its own, maybe we could combine them in some way?

Reply:
Thanks @cybertron i combined into one.